kube/services/jobs/covid19-bayes-cronjob.yaml

```diff
@@ -0,0 +1,53 @@
+# gen3 job run covid19-bayes-cronjob S3_BUCKET <S3_BUCKET>
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: covid19-bayes
+spec:
+  # Weekly Sunday, Wednesday 6:00PM Chicago time == Monday,Tuesday 12:00AM UTC
+  schedule: "0 0 * * 1,4"
+  successfulJobsHistoryLimit: 2
+  failedJobsHistoryLimit: 2
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels:
+            app: gen3job
+        spec:
+          serviceAccountName: s3-access-opencdn-databucket-gen3
+          restartPolicy: Never
+          nodeSelector:
+            role: "jupyter"
+          tolerations:
+            - key: "role"
+              operator: "Equal"
+              value: "jupyter"
+              effect: "NoSchedule"
+          containers:
+            - name: covid19-bayes
+              imagePullPolicy: Always
+              ports:
+              - containerPort: 80
+              GEN3_COVID19-BAYES_IMAGE
+              env:
+              - name: slackWebHook
+                valueFrom:
+                  configMapKeyRef:
+                    name: global
+                    key: slack_webhook
+              - name: gen3Env
+                valueFrom:
+                  configMapKeyRef:
+                    name: manifest-global
+                    key: hostname
+              - name: S3_BUCKET
+                GEN3_S3_BUCKET
+              resources:
+                requests:
+                  cpu: 4
+                  memory: 20Gi
+                limits:
+                  cpu: 4
+                  memory: 20Gi # guess on upper bound
```

Comment:
Let's not give it this much cpu/memory. We can ask @wangfan860 if she knows how much it needs, and if she doesn't, let's use a lower value (maybe [this](https://github.com/uc-cdis/cloud-automation/blob/573c3e83859cb54145b6704104e512f31bd68854/kube/services/jobs/covid19-notebook-etl-job.yaml#L47-L50)) and increase it if we need to

Reply:
my test run was on a 30G, 16cores vm. I noticed that the actual job was using 4cores. for the memory, I would need to double check again to see how much it uses when it's peaked. @nss10 @paulineribeyre